arviz/plots/ppcplot.py

```diff
@@ -110,6 +110,9 @@ def plot_ppc(
         check the plotting method of the backend.
     show : bool, optional
         Call backend show function.
+    group : str, optional
```

Comment:
I would use `group : {"prior", "posterior"}, optional` trying to follow numpydoc style. Also, just to be extra clear, I would only allow two options, prior or posterior.

Reply:
Yeah that makes sense